rest-service/manager_rest/test/endpoints/test_plugins_updates.py

```diff
@@ -0,0 +1,365 @@
+class PluginsUpdates(BaseServerTestCase):
+    @suppress_plugins_change_detection
+    def test_returns_relevant_plugins_updates(self):
+        self.put_file(*self.put_blueprint_args(blueprint_id='hello_world'))
+        self.client.deployments.create('hello_world', 'd123')
+        self.wait_for_deployment_creation(self.client, 'd123')
+        plugins_update1 = self.client.plugins_update.update_plugins(
+            'hello_world')
+        plugins_update2 = self.client.plugins_update.update_plugins(
+            'hello_world', force=True)
+        plugins_update_set = set(p.id for p in self.client.plugins_update
+                                 .list().items)
+        self.assertEqual(2, len(plugins_update_set))
+        self.assertIn(plugins_update1.id, plugins_update_set)
+        self.assertIn(plugins_update2.id, plugins_update_set)
+
+    def test_returns_empty_list_with_no_plugins_updates(self):
+        self.assertEqual(0, len(self.client.plugins_update.list().items))
+
+
+@attr(client_min_version=base_test.LATEST_API_VERSION,
+      client_max_version=base_test.LATEST_API_VERSION)
+class PluginsUpdateId(BaseServerTestCase):
+    def test_raises_when_plugins_update_doesnt_exist(self):
+        with self.assertRaisesRegexp(
+                CloudifyClientError,
+                'Requested `PluginsUpdate` with ID `non_existing` was '
+                'not found') as e:
+            self.client.plugins_update.get('non_existing')
+            self.assertEqual(404, e.exception.status_code)
+
+    @suppress_plugins_change_detection
+    def test_returns_correct_plugins_update(self):
+        self.put_file(*self.put_blueprint_args(blueprint_id='hello_world'))
+        self.client.deployments.create('hello_world', 'd1')
+        self.wait_for_deployment_creation(self.client, 'd1')
+        plugins_update = self.client.plugins_update.update_plugins(
+            'hello_world')
+        response_plugins_update = self.client.plugins_update.get(
+            plugins_update.id)
+        self.assertEqual(plugins_update.id, response_plugins_update.id)
+
+
+@attr(client_min_version=base_test.LATEST_API_VERSION,
+      client_max_version=base_test.LATEST_API_VERSION)
+class PluginsUpdateTest(BaseServerTestCase):
+    """
+    Test plugins update.
+    """
+
+    def setUp(self):
+        super(PluginsUpdateTest, self).setUp()
+        self._sm = get_storage_manager()
+
+    def test_raises_with_nonexisting_blueprint(self):
+        with self.assertRaises(CloudifyClientError) as e:
+            self.client.plugins_update.update_plugins("non_existing")
+            self.assertEqual(404, e.exception.status_code)
+
+    @suppress_plugins_change_detection
+    def test_raises_blueprint_has_no_deployments(self):
+        self.put_file(*self.put_blueprint_args(blueprint_id='hello_world'))
+        with self.assertRaisesRegexp(CloudifyClientError,
+                                     "The blueprint hello_world has no "
```

Comment:
I think it will be more readable to put "" around the blu name

Reply:
I surrounded it with ' '